cookieConsent/OneTrust/browser.js

```diff
@@ -77,7 +74,6 @@ class OneTrust {
 
       return containsAllConsent;
     } catch (e) {
-      logger.error(`Error during onetrust cookie consent management ${e}`);
```

Comment:
This is necessary. Let's keep it

Reply:
Size increasing because of this.

Reply:
Can you find something else to optimize? Worst case we'll increase the limit but cannot compromise on errors